package.json

```diff
@@ -504,6 +504,14 @@
           "type": "string",
           "description": "Color of the search highlight text."
         },
+        "vim.searchMatchColor": {
+          "type": "string",
+          "description": "Color of the current match."
```

Comment:
I think type should be `["string", "undefined"]` and description should mention that it uses `editor.findMatchBackground` if `undefined`